frontend/src/components/pages/LogRecords.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
```

Comment:
Is this frontend code part of Safwaan's PR? I think his changes have been merged to main, can you pull the latest main so that these changes go away from this PR (this PR should only be backend stuff).